libbeat/api/server.go

```diff
@@ -51,16 +56,16 @@ func New(mux *http.ServeMux, config *common.Config) (*Server, error) {
 		return nil, err
 	}
 
-	return &Server{mux: mux, l: l, config: cfg}, nil
+	return &Server{mux: mux, l: l, config: cfg, log: log.Named("API")}, nil
```

Comment:
I've downcased the api.

>  This will create a new logger and append .API to the existing logger's name (if it had a name).

Do you find the append problematic?

Not sure adding the listener or endpoint is useful, we only have a single responder?